src/data_source/storage/sql/transaction.rs

```diff
@@ -274,117 +283,37 @@ impl_tuple_params!(6, (T1, T2, T3, T4, T5, T6,));
 impl_tuple_params!(7, (T1, T2, T3, T4, T5, T6, T7,));
 impl_tuple_params!(8, (T1, T2, T3, T4, T5, T6, T7, T8,));
 
-impl<'p, T> Params<'p> for Vec<T>
+pub fn build_where_in<'a, I>(
+    column: &'a str,
+    query: &'a str,
+    values: I,
```

Comment:
Nit: it makes more sense to me if we shuffle these arguments so it's `query`, `column`, `values`, because that's the order you would write it in if you were writing out the SQL: `SELECT stuff WHERE {column} IN {values}`